clang/lib/CIR/CodeGen/CIRGenFunction.h

```diff
@@ -110,6 +110,8 @@ class CIRGenFunction : public CIRGenTypeCache {
 public:
   mlir::Value createDummyValue(mlir::Location loc, clang::QualType qt);
 
+  void emitNullInitialization(mlir::Location loc, Address destPtr, QualType ty);
```

Comment:
The two emit functions you've added are still not in the correct place, but that's fine. Some others have gotten out of order too, so I'll make another reordering patch after this is merged.

Reply:
Thank you, Currently I have time if you prefer, I can handle it